corehq/apps/userreports/templates/userreports/partials/link_reports.html

```diff
@@ -0,0 +1,38 @@
+{% load i18n %}
+{% load hq_shared_tags %}
+{% load crispy_forms_tags %}
+<button type="button"
+        data-toggle="modal"
+        data-target="#link-reports"
+        class="btn btn-default">
+  <i class="fa fa-link"></i> {% trans "Copy Report" %}
+</button>
+
+<div class="modal fade" id="link-reports">
+  <div class="modal-dialog">
+    <div class="modal-content">
+      <div class="modal-header">
+        <button type="button" class="close" data-dismiss="modal" aria-hidden="true">&times;</button>
+        <h3>
+          {% trans "Copy Report" %}
+        </h3>
+      </div>
+      <div class="modal-body">
+        {% if copy_report_form %}
```

Comment:
Why wouldn't `copy_report_form` exist - and if it doesn't, shouldn't the button to trigger the modal be hidden?